### DISCUSSION TITLE
Gosund

### DISCUSSION THREAD

**Discussion by quorea** [milo526/homebridge-tuya-web#523] [General]
Hello guys,
Thanks for your help and ingenuity. Does this plugin support Gosund? I have Gosund. branded product and use the Gosund app which is also based on Tuya and struggling to integrate them to HK. Any help?

**Comment by JeffGoldner** [comment]
Yes, I have a Gosund 6 outlet+3 USB plug and I use this plugin. 3 outlets and 1 USB switched. I use the Smart Life app, and have selected Smart Life as the platform in the TuyaWebPlatform plug-in settings.
